src/main/java/org/eclipse/tractusx/ssi/lib/exception/key/InvalidPrivateKeyFormatException.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * ******************************************************************************
+ * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * *******************************************************************************
+ */
+
+package org.eclipse.tractusx.ssi.lib.exception.key;
+
+/** The type Invalide private key format. */
+public class InvalidPrivateKeyFormatException extends KeyGenerationException {
+  private static final long serialVersionUID = -3348735256693555408L;
```

Comment:
Add the `@Serial` annotation
Make it equal `1L`. as it's a new class
Repeat for all new Exception classes.

Reply:
I think we need to use this compilation option `javac -Xlint:serial` to make it work, so I didn't change.

Reply:
No, it's actually preferable to start at `1L` and increment only when you've changed the class, in order to ensure serialization doesn't break between compilers, platforms and other outside factors. For exceptions it is sufficient to set it at 1L